packages/appstore/src/components/currency-switcher-card/demo/demo-account-card.tsx

```diff
@@ -6,16 +6,34 @@ import BalanceText from 'Components/elements/text/balance-text';
 import { useStores } from 'Stores/index';
 import './demo-account-card.scss';
 import { localize } from '@deriv/translations';
+import {
+    useCFDDemoAccounts,
+    useCurrencyExchangeRate,
+    usePlatformDemoAccount,
+    useTotalAccountBalance,
+} from '@deriv/hooks';
 
 const DemoAccountCard = () => {
     const { client, traders_hub } = useStores();
-    const { accounts, loginid, resetVirtualBalance } = client;
-    const { platform_demo_balance, selected_account_type } = traders_hub;
+    const { accounts, loginid, resetVirtualBalance, default_currency } = client;
+    const { selected_account_type } = traders_hub;
 
     const canResetBalance = () => {
         return accounts[loginid]?.balance !== 10000;
     };
 
+    const platform_demo_account = usePlatformDemoAccount();
+    const cfd_demo_accounts = useCFDDemoAccounts();
+    const cfd_demo_balance = useTotalAccountBalance(cfd_demo_accounts);
```

Comment:
How does this cater for multiple demo accounts such as MT5 demo and Deriv X demo?
I can see that in `useTotalAccountBalance` you only access to the first `account[0]`

Reply:
actually we get the first index only to get the user's currency. but for calculations it loops over the accounts array passed to the hook.
and as I checked the calculations for multiple accounts is calculated correctly : 
![image](https://user-images.githubusercontent.com/100833613/230883863-184f7080-683d-4cb3-8f2a-a4c54b1e2c8b.png)

if you think I missed anything , please let me know :pray: